atomfeed-console/README.md

```diff
@@ -0,0 +1,13 @@
+# atomfeed-console
+
+This directory contains resources for building Docker image for atomfeed-console. The atomfeed-console container is used to show the failed events that happened in an atomfeed client.
+
+Atomfeed console is  is useful when you are facing some sync issues with components like OpenMRS, OpenELIS, Odoo, etc. Issues can be like patients not getting created, orders not getting synced, pacs orders not getting synced etc.,
+
+## Building the image locally
```

Comment:
Should the build instructions not moved to atomfeed-console repo?

Reply:
@angshu  these build instructions are for building the Docker image and not the atomfeed-console artifact in particular.